### ISSUE TITLE
Display the Price with QR Code

### ISSUE THREAD

**Issue by Steampowered-hippo** [ZeusLN/zeus#1600]
### Describe the feature

When the QR-Code - page of a lightning invoice is shown, the invoice's value should also be displayed in Sats and Fiat. Especially when paying with NFC, you have now no chance to check the value of the invoice.

**Comment by myxmaster** [comment]
I like this idea. But you should be aware that it is easy for scammers to present a QR code to you with a fake amount. So you still need to trust the person that is presenting the QR to you.

**Comment by Steampowered-hippo** [comment]
Yes, I understand the problem. But when you pay via NFC Card you always will have this problem.
I experienced the following: In a restaurant, that accepts BTC via LN, the waiter types the amount usual without the decimal in his (Fiat) cash system (e.g.: he types "450 for 4.50 Fiat). So he wanted me to pay 450 instead of 4.50. When I payed with my own Lightning Wallet (Zeus), the amount was displayed on my device, so I discovered the error. With NFC, I would have had no chance. (aside from a limit). 

Another idea is, to configure a PIN with the card, that I have to type in with the payment. Similar to a payment with Credit card.
